telemetry/options/config.go

```diff
@@ -0,0 +1,9 @@
+package options
+
+type Config struct {
+	MetricsEnabled   bool
+	UseFakeMetrics   bool
```

Comment:
What do you think about completely dropping this param?
I think it's fine for the showcase of the PR, but see no real use for it in a live node

Reply:
dropped